api/src/pcapi/core/mails/transactional/bookings/user_warning_after_pro_booking_cancellation.py

```diff
@@ -0,0 +1,84 @@
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.models.feature import FeatureToggle
+from pcapi.utils.mailing import format_booking_hours_for_email
+from pcapi.utils.mailing import get_event_datetime
+
+
+def get_warn_user_after_pro_booking_cancellation_email_data(
+    booking: Booking,
+) -> Union[dict, SendinblueTransactionalEmailData]:
+    stock = booking.stock
+    offer = stock.offer
+    is_online: bool = offer.isDigital
+    if is_online:
+        is_event = False
+        is_thing = False
+    else:
+        is_event: bool = offer.isEvent
+        is_thing: bool = offer.isThing
+
+    if is_event:
+        event_date = format_date(get_event_datetime(stock), format="full", locale="fr")
+        event_hour = format_booking_hours_for_email(booking)
+    else:
+        event_date = None
+        event_hour = None
+
+    is_free_offer: bool = True if stock.price == 0 else False
+    offerer_name = offer.venue.managingOfferer.name
+    offer_name = offer.name
+    offer_price = str(booking.total_amount)
+    user_first_name = booking.firstName
+    user_last_name = booking.lastName
+    venue_name = offer.venue.publicName if offer.venue.publicName else offer.venue.name
+
+    if not FeatureToggle.ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS.is_active():
+        return {
+            "MJ-TemplateID": 1116690 if booking.individualBooking is not None else 3192295,
+            "MJ-TemplateLanguage": True,
+            "Vars": {
+                "event_date": event_date if event_date else "",
+                "event_hour": event_hour if event_hour else "",
+                "is_event": int(is_event),
+                "is_free_offer": int(is_free_offer),
+                "is_online": int(is_online),
+                "is_thing": int(is_thing),
+                "offer_name": offer_name,
+                "offer_price": offer_price,
+                "offerer_name": offerer_name,
+                "user_first_name": user_first_name,
+                "user_last_name": user_last_name,
+                "venue_name": venue_name,
+            },
+        }
+
+    return SendinblueTransactionalEmailData(
+        template=TransactionalEmail.PRO_BOOKING_CANCELLATION.value,
+        params={
+            "EVENT_DATE": event_date,
+            "EVENT_HOUR": event_hour,
+            "IS_EVENT": is_event,
+            "IS_FREE_OFFER": is_free_offer,
+            "IS_ONLINE": is_online,
+            "IS_THING": is_thing,
+            "OFFER_NAME": offer_name,
+            "OFFER_PRICE": offer_price,
+            "OFFERER_NAME": offerer_name,
+            "USER_FIRST_NAME": user_first_name,
+            "USER_LAST_NAME": user_last_name,
+            "VENUE_NAME": venue_name,
+        },
+    )
+
+
+def send_warning_to_user_after_pro_booking_cancellation_email(booking: Booking) -> None:
+    data = get_warn_user_after_pro_booking_cancellation_email_data(booking)
+    mails.send(recipients=[booking.email], data=data)
```

Comment:
```suggestion
    return mails.send(recipients=[booking.email], data=data)
```

Dans une [autre PR](https://github.com/pass-culture/pass-culture-main/pull/638), on va systématiquement transmettre la valeur de `mails.send()`. On peut faire la même chose ici.

Reply:
done.